docs/.vuepress/sidebar/en.ts

```diff
@@ -29,7 +30,7 @@ export const enSidebar = sidebar({
     },
     {
       text: "Tutorials",
-      link: "/dev/tutorials/cross-chain-tutorial.md",
+      link: "/dev/tutorials/README.md",
```

Comment:
Can you do the same for the "How to". That is, add a README with summary.

Reply:
fixed in https://github.com/matter-labs/zksync-web-era-docs/pull/625/commits/be942f012e5c9caa23056d437a54f1babb8818c0